pkg/controller/template/render_test.go

```diff
@@ -31,9 +31,6 @@ func TestCloudProvider(t *testing.T) {
 		featureGate *configv1.FeatureGate
 		res         string
 	}{{
-		platform: configv1.AWSPlatformType,
```

Comment:
I don't fully understand. Are these no longer in use i.e. cloudProvider ID will be only set using featuregate?

Reply:
Just moved this case to line 61.